docs/sources/concepts/index.md

```diff
@@ -0,0 +1,75 @@
+---
+description: Learn about concepts basic to profiling.
+keywords:
+  - Explore Profiles
+  - Concepts
+title: Concepts
+menuTitle: Concepts
+weight: 200
+---
+
+# Concepts
+
+To use profiling, you should understand these concepts:
+
+* Profile data hierarchy
+* Profile types
+* Labels
+* Flame graphs
+
+## Profile data hierarchy
+
+Each profile has data for services. A service can be one of the processes running on your system or application.
+
+Each service has multiple profile types, such as CPU, memory, exceptions, and locks. These profiles provide details about processes within a service.
+
+* Service:  This is the main application or system you are monitoring. For example, it could be an online shopping platform, a social media application, or a cloud storage service
```

Comment:
```suggestion
* Service:  This is the main application or system you are monitoring. For example, it could be an online shopping platform, a social media application, or a cloud storage service.
```